### PR TITLE
Add standard (master) repo subscription

### PR DESCRIPTION
**Pull request by chcosta**
@dagood  PTAL

/cc @weshaggard

### CODE REVIEW

**Comment by chcosta** [comment]
Also, I don't think I have permissions to merge this, so @dagood , can you please merge pending review?

File: Maestro/subscriptions.json [hunk 1]
```diff
@@ -739,6 +745,30 @@
...
+    // Update dependencies in Standard master
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/buildtools/master/Latest.txt"
+      ],
+      "action": "standard-general",
+      "delay": "00:10:00",
+      "actionArguments": {
+        "vsoBuildParameters": {
+          "ScriptFileName": "build-managed.cmd",
```
Comment (dagood) [inline Maestro/subscriptions.json]: I think this should just be `build.cmd` for standard--I see a `build-managed.sh` but no `build-managed.cmd`.

**Comment by chcosta** [comment]
@dagood , as far as you are aware, after this merges, is there any other items required to be done for standard to be performing maestro updates of buildtools changes?

**Comment by dagood** [comment]
This should be the last bit. We'll know for sure after the next BuildTools build.

File: Maestro/subscriptions.json [hunk 2]
```diff
@@ -739,6 +745,30 @@
...
     },
+    // Update dependencies in Standard master
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/buildtools/master/Latest.txt"
+      ],
+      "action": "standard-general",
+      "delay": "00:10:00",
+      "actionArguments": {
+        "vsoBuildParameters": {
```
Comment (dagood) [inline Maestro/subscriptions.json]: I didn't notice that the build def's default branch is `dev/release/2.0.0`--this made the first auto-update attempt do nothing: https://devdiv.visualstudio.com/DevDiv/_build/index?buildId=1089726. `actionArguments` either needs a `vsoSourceBranch`, or the def needs to default to `master`.

To be sure, I ran the auto-PR target with my bot on master and it worked fine: https://github.com/dagood/standard/pull/1.

Comment (chcosta) [reply Maestro/subscriptions.json]: I updated the default to master, and got a weird error.

https://devdiv.visualstudio.com/DevDiv/_build/index?buildId=1089926&_a=summary

Actually, the job didn't report an error, but it clearly did not do anything expected...

```
2017-10-26T16:14:08.5008059Z ScriptFileName : The term 'ScriptFileName' is not recognized as the name of a cmdlet, function, script file, or 
2017-10-26T16:14:08.5008059Z operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try 
2017-10-26T16:14:08.5008059Z again.
2017-10-26T16:14:08.5008059Z At line:1 char:82
2017-10-26T16:14:08.5008059Z + ... 47ea-971b-6737f3faa038.ps1' -ScriptFileName "$(ScriptFileName)" -Argu ...
2017-10-26T16:14:08.5008059Z +                                                    ~~~~~~~~~~~~~~
2017-10-26T16:14:08.5008059Z     + CategoryInfo          : ObjectNotFound: (ScriptFileName:String) [], CommandNotFoundException
2017-10-26T16:14:08.5008059Z     + FullyQualifiedErrorId : CommandNotFoundException
2017-10-26T16:14:08.5008059Z  
```

Have you seen this before?  I don't see what is different between this definition and the definition for other branches.

Comment (dagood) [reply Maestro/subscriptions.json]: The issue is trying to queue it with defaults: that doesn't work. :smile: If you pass `ScriptFileName` as defined in the json and join `Arguments` with spaces and pass that in as `Arguments`, that should work.

Comment (chcosta) [reply Maestro/subscriptions.json]: Ah, I see.  Silly that there's no way to manually queue a job with defaults, but I'll create a PR.

Comment (dagood) [reply Maestro/subscriptions.json]: Yeah, awkward to do end-to-end runthroughs. This is tracked in https://github.com/dotnet/versions/issues/99.

Is the PR to add in a branch name? If you changed the default to master, I don't think any PR is required.

Comment (chcosta) [reply Maestro/subscriptions.json]: Yes, that's the PR.  I guess it's overkill, but also, probably slightly safer to be explicit in the subscription.  I can close the PR if you'd prefer though.

Comment (dagood) [reply Maestro/subscriptions.json]: Cool, agreed on being explicit, will make it less error-prone to fork off release branch subscriptions for example.

**Pull request merged**